cmd/crossplane/main.go

```diff
@@ -48,12 +53,26 @@ func main() {
 		// logger when we're running in debug mode.
 		ctrl.SetLogger(zl)
 	}
+	// Note that the controller managers scheme must be a superset of the
+	// package manager's object scheme; it must contain all object types that
+	// may appear in a Crossplane package. This is because the package manager
+	// uses the controller manager's client (and thus scheme) to create packaged
+	// objects.
+	s := runtime.NewScheme()
+	utilruntime.Must(scheme.AddToScheme(s))
+	utilruntime.Must(extv1.AddToScheme(s))
+	utilruntime.Must(extv1beta1.AddToScheme(s))
+	utilruntime.Must(apis.AddToScheme(s))
```

Comment:
Does `utilruntime.Must` panic if these `AddToScheme` calls fail? Could we use `kingpin.FatalIfError` instead? I'd prefer to be consistent with our existing patterns.